vtex/ab-tester
# Fix/vbase client

This PR aims to adapt the last change we made in `ab-tester` evolving how it stores the data about previous tests in a way that allow us to use it also as an index to the the files with test's metadata. Also it adds some logs for possible errors when trying to read data from VBase.

The path we're following is one that allow us to don't delete anymore the metadata about the tests and for that we want to create some index that can be searched to find what is the current test, if is there one.

The main change relates to the way we store the test results. Now we'll save the index as an interface - which is a format we could evolve now and maybe in the future if we want - and then stores there the array with previous tests and also a field that says what is the current test being executed.

#### How should this be manually tested?
We should try to initialize and finish an A/B Test and in every state-transition we should check the files where we save the metadata to verify if they are changed accordingly to what we expect.

#### Types of changes
